graylog2-server/src/test/java/org/graylog2/database/utils/MongoUtilsTest.java

```diff
@@ -46,14 +46,19 @@ class MongoUtilsTest {
 
     private record DTO(@Id @org.mongojack.ObjectId String id, String name) implements MongoEntity {}
 
+    private record DTORef(@Id @org.mongojack.ObjectId String id, @org.mongojack.ObjectId ObjectId refId,
```

Comment:
I think the `@ObjectId` annotation only makes sense for fields of type `String`?

Reply:
Yes, copy'n'waste mistake, I'll fix

Reply:
Actually, I think it's correct (note it's the same on the other record):
Per mongojack ObjectId javadoc
```
Indicates that the property should be converted to an ObjectId before being stored in the database, and back to whatever type it is afterwards. Use this if you want an ID to have a String or byte[] value, but you want it stored in the database as an ObjectId.
```

Reply:
Nevermind, confused the parameters :)